src/ui/mainwindow.cpp

```diff
@@ -216,6 +216,7 @@ MainWindow::MainWindow(Application* app, SystemTrayIcon* tray_icon, OSD* osd,
   int volume = app_->player()->GetVolume();
   ui_->volume->setValue(volume);
   VolumeChanged(volume);
+  app_->player()->ReloadSettings();
```

Comment:
Why doing this here?
Probably to init menu_previousmode_ value?
If so, it might be better to init it in Player's constructor (maybe by calling ReloadSettings? Not sure if there are other places where we do something similar).